projects/word-weaver/src/assets/i18n/en.json

```diff
@@ -184,7 +184,7 @@
           "random": {
             "pers": {
               "intransitive": "Random pronoun '{{ value }}' selected",
-              "transitive": "Random pronoun '{{ agent }} > {{ patient }}' selected"
+              "transitive": "Random pronouns '{{ agent }} > {{ patient }}' selected"
```

Comment:
ah - this is true for English and French - there are two pronouns for a transitive verb, but in Kanyen'kéha there is only one pronoun for a transitive verb (the subject/object are fused). I suppose this makes sense here, but we'll have to remember to change it in the Kawennón:nis instance.